### PR TITLE
[PNA] Update rollout timeline.

### PR DESCRIPTION
**Pull request by letitz**
PNA warning preflights are rolling out in Chrome 102 once more. Enforcement will be enabled in 105 at the earliest. In general, timelines have been set back by 4 milestones.

https://deploy-preview-2714--developer-chrome-com.netlify.app/blog/private-network-access-preflight/

### CODE REVIEW

**Comment by letitz** [comment]
@agektmr for review.

**Review by agektmr — APPROVED** [review]
LGTM

**Comment by agektmr** [comment]
I'd like a tech writer eyes on this. cc: @heyawhite

File: site/en/blog/private-network-access-preflight/index.md [hunk 1]
```diff
@@ -67,6 +63,10 @@ the change and adjust accordingly.
       the same time to allow for websites affected by this phase to request a
       time extension. The trial will last for at least 6 months.
 
+An earlier attempt was made at rolling out warnings in Chrome 98, as previously
+announced by this blog post. This was rolled back after stability and
+compatibility issues were discovered during the rollout. The identified issues
+were then fixed before the second attempt in Chrome 102.
```
Comment (heyawhite) [inline site/en/blog/private-network-access-preflight/index.md:69]:
```suggestion
{% Aside %}
An earlier attempt was made to roll out warnings in Chrome 98, previously
announced by this blog post. This was rolled back after stability and
compatibility issues were discovered during the rollout.

The identified issues were fixed for Chrome 102.
{% endAside %}
```

**Review by heyawhite — APPROVED** [review]
Seeing as @agektmr approved the timelines, I've made a copy clarification that I'll accept and then merge the updates

**Pull request merged**